keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -398,12 +450,23 @@ class TitusClusterHandler(
         } ?: job
       }
 
+  private fun ResourceDiff<TitusServerGroup>.willDeployNewVersion(): Boolean =
+    !isCapacityOnly() && !isEnabledOnly()
+
   /**
    * @return `true` if the only changes in the diff are to capacity.
    */
   private fun ResourceDiff<TitusServerGroup>.isCapacityOnly(): Boolean =
     current != null && affectedRootPropertyTypes.all { it == Capacity::class.java }
 
+  /**
+   * @return true if the only difference is in the onlyActiveServerGroup property
+   */
+  private fun ResourceDiff<TitusServerGroup>.isEnabledOnly(): Boolean =
```

Comment:
nit: consider naming it `isActiveOnly()` for consistency with the naming elsewhere.

Reply:
Active has a weird meaning that I don't really want to conflate here.

Reply:
OK. In that case I don't know what "enabled" means in this context, haha. 🤷‍♂️

Reply:
Ok, let me explain the underlying weirdness:
- clouddriver has an endpoint for getting the "active" server group, which for us is basically the latest enabled server group.
- that meaning of "active" is used throughout here
- this property I guess could be called "onlyEnabledServerGroup" - maybe that would be clearer? probably would be a lot clearer

If I rename to `onlyEnabledServerGroup` for the property, does that make more sense?

Reply:
It does, but don't bother, it's alright.